civictechprojects/apps.py

```diff
@@ -4,7 +4,9 @@
 class CivictechprojectsConfig(AppConfig):
     name = 'civictechprojects'
 
-    def ready(self):
+    # def ready(self):
         # Remove any tags that aren't in the canonical tag list
-        from .models import Project
-        Project.remove_tags_not_in_list()
+        # TODO: Fix so this doesn't break in production database
```

Comment:
Add a pointer to the tracking task for this work item?

Reply:
Edited the comment.  Not sure exactly what work we should do for this yet, so holding off on a tracking task for now.